ArgoTests/Tests/PerformanceTests.swift

```diff
@@ -13,9 +13,9 @@ class PerformanceTests: XCTestCase {
   func testDecodePerformance() {
     let json: AnyObject = JSONFromFile("big_data")!
     let j = JSON.parse(json)
-
+    
```

Comment:
Trailing Whitespace Violation: Lines should not have trailing whitespace. (trailing_whitespace)